cmake/AstarteMQTTTransport.cmake

```diff
@@ -22,6 +22,7 @@ function(astarte_sdk_configure_mqtt_dependencies)
         find_package(cpr REQUIRED)
         find_package(nlohmann_json REQUIRED)
         find_package(MbedTLS REQUIRED)
+        find_package(stduuid REQUIRED)
```

Comment:
Is it possible to bind some dependencies only if required?
Only if we want MQTT feature should add `MbedTLS`, `stduuid` etc...
Let's discuss it IRL.